linux_os/guide/services/ldap/openldap_client/ldap_client_start_tls/rule.yml

```diff
@@ -32,7 +32,7 @@ references:
     stigid@rhel6: "000252"
     disa@rhel6: 776,778
     disa: "1453"
-    nist: AC-17(a),AC-17(2),CM-7
+    nist: AC-17(a),AC-17(2),CM-7,SC-12(3)
```

Comment:
Since RHEL's TLS implementation underwent FIPS, would SC-12(2) be a better fit?


SC-12(2) | CRYPTOGRAPHIC KEY ESTABLISHMENT AND MANAGEMENT \| SYMMETRIC KEYSThe organization produces, controls, and distributes symmetric cryptographic keys using [Selection: NIST FIPS-compliant; NSA-approved] key management technology and processes.
-- | --

Reply:
@shawndwells doesn't TLS use both asymmetric and symmetric keys? So, isn't the answer both?